//
// PROJECT:         Aspia Remote Desktop
// FILE:            proto/auth_session.proto
// LICENSE:         See top-level directory
// PROGRAMMERS:     Dmitry Chapyshev (dmitry@aspia.ru)
//

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

import "auth_session_message.proto";
import "status.proto";

package aspia.proto.auth;

message HostToClient
{
    SessionType session_type = 1;
    Status status            = 2;
}

message ClientToHost
{
    SessionType session_type = 1;
    AuthMethod method        = 2;
    string username          = 3;
    bytes password           = 4;
}
